osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -0,0 +1,356 @@
+                                                        Text = HumanizerUtils.Humanize(comment.CreatedAt)
+                                                    },
+                                                    new RepliesButton(comment.RepliesCount)
+                                                    {
+                                                        Expanded = { BindTarget = childrenExpanded }
+                                                    },
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    childCommentsVisibilityContainer = new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                        Children = new Drawable[]
+                        {
+                            childCommentsContainer = new FillFlowContainer
+                            {
+                                Padding = new MarginPadding { Left = 20 },
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Vertical
+                            },
+                            deletedChildrenPlaceholder = new DeletedChildrenPlaceholder
+                            {
+                                ShowDeleted = { BindTarget = ShowDeleted }
+                            }
+                        }
+                    }
+                }
+            };
+
+            deletedChildrenPlaceholder.DeletedCount.Value = comment.GetDeletedChildrenCount;
+
+            if (comment.UserId.HasValue)
+                username.AddUserLink(comment.User);
+            else
+                username.AddText(comment.LegacyName);
+
+            if (comment.EditedAt.HasValue)
+            {
+                info.Add(new SpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Font = OsuFont.GetFont(size: 12),
+                    Text = $@"edited {HumanizerUtils.Humanize(comment.EditedAt.Value)} by {comment.EditedUser.Username}"
+                });
+            }
+
+            if (!comment.IsDeleted)
+            {
+                var formattedSource = MessageFormatter.FormatText(comment.GetMessage());
+                message.AddLinks(formattedSource.Text, formattedSource.Links);
+            }
+            else
+            {
+                content.FadeColour(OsuColour.Gray(0.5f));
+                votePill.Hide();
+            }
+
+            if (comment.IsTopLevel)
```

Comment:
What about for comments nested multiple levels deep? Example: https://osu.ppy.sh/comments/915348

Reply:
It works, as you can see in tests
![Screenshot_2](https://user-images.githubusercontent.com/22874522/66813235-14a65780-ef3d-11e9-8179-7bf7e7b24494.png)